razzle.config.js

```diff
@@ -96,7 +97,20 @@ const defaultModify = ({
         name: dev,
       },
     });
-
+    if (process.env.NODE_ENV === 'production') {
+      config.plugins.push(
+        new OfflinePlugin({
+          caches: {
+            main: [':rest:'],
```

Comment:
@tiberiuichim I think we can alter 'what modules to cache' and make it configurable via settings?

Reply:
What would that look like?

Reply:
```js
main: [settings.cachedAssets || ":rest:"]
```
Here `settings.cachedAssets` can be provided as a set of pattern matching and it should contain the most important assets, not all. 
```cachedAssets: ["main.js", "main-2.js"]```
The other thing can be caching onRoute. This can create more dynamic sense of caching strategy for us.

Reply:
or let the user to exclude non-critical assets?

Reply:
We don't have access to settings inside razzle.config.js